### PR TITLE
Initial Kinesalite docker image

### PR DESCRIPTION
**Pull request by mikemoate**
A docker image to provide Kinesalite for development and testing, in lieu of AWS Kinesis Streams.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM node:8
```
Comment (randomjunk) [inline Dockerfile]: any reason not `node:8-alpine` here? (just in the interests of smaller image size)

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+RUN groupadd kinesalite && useradd --gid kinesalite kinesalite
+
+RUN mkdir /var/kinesalite
+RUN chown kinesalite /var/kinesalite
+
+USER kinesalite
+
+EXPOSE 4567
+ENTRYPOINT ["kinesalite", "--path", "/var/kinesalite"]
```
Comment (randomjunk) [inline Dockerfile]: Do we need to specify path here?
If the primary purpose of the library is correctness testing I wouldn't have thought there would be much data passing through, and the keep-in-RAM mode would be faster.

Path can always be added at the run stage if people need it.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
+# Kinesalite
+
+[![Docker Automated buil](https://img.shields.io/docker/build/countingup/kinesalite.svg)](https://hub.docker.com/r/countingup/kinesalite/builds/)
```
Comment (randomjunk) [inline README.md]: missing buil`d` ?

**Review by randomjunk — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (mikemoate) [reply Dockerfile]: Now done, though was more challenging than expected...

File: Dockerfile (hunk 2 above)
Comment (mikemoate) [reply Dockerfile]: No and fair point, removed.

**Review by randomjunk — APPROVED** [review]
LGTM

**Pull request merged**